README.md

```diff
@@ -42,6 +42,25 @@ hooks('ts').push(function({content}){
 /* transpile your script code here */
 });
 ```
+
+For custom blocks in `.vue` files, add other require hooks for the name of the blocks,
+prefixed by `vue-block-`. For example, to load a custom `<json></json>` block:
+
+```javascript
+const hooks = require('require-extension-hooks');
+hooks('vue').plugin('vue');
+hooks('vue-block-json').push(({ content }) => {
+    // Strings returned here will be appended to the compiled .vue file. This should return JavaScript, in a string.
+    return '';
+});
+```
+
+You can access the exported options object using (as vue-template-compiler does itself):
+```javascript
+((module.exports.default || module.exports).options || module.exports.default || module.exports)
```

Comment:
I'm not overly keen on having this example in the docs as it's not pretty. Instead could you replace this whole paragraph with an explanation that the returned content will be appended to the file